context.go

```diff
@@ -96,6 +96,20 @@ func (c *Context) RunScript(source string, origin string) (*Value, error) {
 	return valueResult(c, rtn)
 }
 
+// RunScript executes the source JavaScript; origin or filename provides a
+// reference for the script and used in the stack trace if there is an error.
+// error will be of type `JSError` of not nil.
+func (c *Context) RunCompiledScript(source string, compiledSource []byte, origin string) (*Value, error) {
+	cSource := C.CString(source)
+	cOrigin := C.CString(origin)
+	defer C.free(unsafe.Pointer(cSource))
+	defer C.free(unsafe.Pointer(cOrigin))
+
+	rtn := C.RunCompiledScript(c.ptr, cSource, (*C.uchar)(unsafe.Pointer(&compiledSource)), C.int(len(compiledSource)), cOrigin)
```

Comment:
We don't want a pointer to the slice itself.  That would be a pointer to a struct like `typedef struct { void *data; GoInt len; GoInt cap; } GoSlice;`.  We need to make sure we get a pointer to the data, which can be done in Go by getting a reference to the first element

```suggestion
	rtn := C.RunCompiledScript(c.ptr, cSource, (*C.uchar)(unsafe.Pointer(&compiledSource[0])), C.int(len(compiledSource)), cOrigin)
```

This fixed the issue with the code cache being rejected that you mentioned in direct messages.